spaces/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Address(models.Model):
+    street_name = models.CharField(max_length=140)
+    civic = models.CharField(max_length=20)
```

Comment:
And, as that very page reads, it seems not every country uses the same numbering system, and many of them are not like ours.